packages/storybook-html/src/components/footer/footer.html

```diff
@@ -41,9 +41,11 @@
                         </ul>
                     </div>
                 </div>
+                <!-- 
```

Comment:
Can we include a note regarding why this is commented out, or remove it from the footer, include it as a seperate example?